provision-osx/defaults/main.yml

```diff
@@ -59,6 +59,7 @@ homebrew_taps:
 
 xcode_versions:
 - '8.3.3'
+- '9'
```

Comment:
Are we automatically purging unlisted versions ? If no, I would prefer defaulting just o latest - remove 8.3.3. Wdyt ?

Reply:
Nope, older versions are not purged at all.

Sure lets keep the latest only.